src/test/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretTest.java

```diff
@@ -17,11 +15,11 @@
 @DataJpaTest
 public class TeeTaskComputeSecretTest {
 
-    @Autowired
-    private EntityManager entityManager;
+    private final TeeTaskComputeSecretRepository teeTaskComputeSecretRepository;
 
-    @Autowired
-    private TeeTaskComputeSecretRepository teeTaskComputeSecretRepository;
+    TeeTaskComputeSecretTest(@Autowired TeeTaskComputeSecretRepository teeTaskComputeSecretRepository) {
```

Comment:
Side question, did you notice any difference in **tests** between using Autowired in constructor vs field?

Reply:
I did not notice a difference. I remembered it was a possibility and it seems to work well with linters.

Reply:
Ok, personally I usually go for `@Autowired` on fields for tests and `autowired` injection in contructors of app services (cause I've already seen issues with direct annotation on fields)

Reply:
I will keep this in mind and try to remember it if I face a weird behavior in the future.